kusto/data/value/decimal.go

```diff
@@ -32,7 +32,7 @@ func (d *Decimal) ParseFloat(base int, prec uint, mode big.RoundingMode) (f *big
 	return big.ParseFloat(d.Value, base, prec, mode)
 }
 
-var decRE = regexp.MustCompile(`^\d*\.\d+$`)
+var DecRE = regexp.MustCompile(`^((\d+\.?\d*)|(\d*\.?\d+))$`) // Matches decimal numbers, with or without decimal dot, with optional parts missing.
```

Comment:
This is the one I used in my fork: `^\d*\.\d+$|^\d+$`

Reply:
Sadly it doesn't grab the legitimate case of "1."